input/pagecontent/examples.md

```diff
@@ -15,7 +15,13 @@ In addition to the examples defined in this implementation, synthetic (realistic
 * Individual Pronouns: [Patient - Mia Leanne Banks](Patient-banks-mia-leanne.html)
 * Gender Identity: [Patient - Mia Leanne Banks](Patient-banks-mia-leanne.html)
 
-
+### Missing Data & Suppressed Data
+* Missing Data:
+  * [Patient - Ronny Irvine (missing data for birth date)](Patient-ronny-irvine.html)
```

Comment:
The presentation of elements that are missing is different between Missing Data and Suppressed Data. Suggest that missing data presentation is revised.